olp-cpp-sdk-core/include/olp/core/logging/Log.h

```diff
@@ -33,6 +33,21 @@
 #include <sstream>
 #include <string>
 
+/**
+ * @brief Temporary macros to ease EDGE_SDK_ prefix transition review.
+ */
+#define LOG_FUNCTION_SIGNATURE EDGE_SDK_LOG_FUNCTION_SIGNATURE
```

Comment:
you need to move these down as last elements in the file else the macros will not expend correctly.

Reply:
https://gcc.gnu.org/onlinedocs/cpp/Argument-Prescan.html#Argument-Prescan
> Macro arguments are completely macro-expanded before they are substituted into a macro body, unless they are stringized or pasted with other tokens. After substitution, the entire macro body, including the substituted arguments, is scanned again for macros to be expanded. The result is that the arguments are scanned twice to expand macro calls in them.

In short, the macros will be expanded correctly. I'd prefer to keep them at the top of the file to catch an eye better.